sycl/include/CL/sycl/handler.hpp

```diff
@@ -1294,6 +1294,7 @@ class __SYCL_EXPORT handler {
   /// \param KernelFunc is a SYCL kernel function.
   template <typename KernelName = detail::auto_name, typename KernelType,
             int Dims>
+  __SYCL2020_DEPRECATED("offsets are deprecated in SYCL2020")
```

Comment:
can we say what to use instead?

Reply:
Nothing. There's no replacement in SYCL2020.